content/applications/inventory_and_mrp/manufacturing/basic_setup/mo_costs.rst

```diff
@@ -31,7 +30,8 @@ Cost configuration
 
 Odoo computes |MO| costs based on the configuration of the |BoM| used to manufacture a product. This
 includes the cost and quantity of components and operations listed on the |BoM|, in addition to the
-operating costs of the work centers where those operations are carried out.
+operating costs of the work centers where those operations are carried out, and the amount paid to
+each employee who works on an operation.
```

Comment:
The admonition block applies to the whole doc, while the intro of the first section is only relevant to that section. I think it makes sense to reiterate the information since it's kind of confusing and some readers might skip the admonition block.